corehq/apps/userreports/tasks.py

```diff
@@ -10,7 +10,7 @@
 from dimagi.utils.couch.database import iter_docs
 
 
-@task(queue='background_queue', ignore_result=True)
+@task(queue='background_queue', ignore_result=True, acks_late=True)
```

Comment:
I think it's fine, we just need to keep in mind that if we ever change brokers we need to revisit all of our acks_late tasks and make sure things will still work the way we want them to.